pgml-docs/docs/gym/fraud.md

```diff
@@ -0,0 +1,460 @@
+Risk mitigation
+===============
+
+Most organizations have some risks that may be minimized using machine learning, by predicting the likelihood of negative outcomes before they happen. As long as you're able to track the information leading up to the unfortunate events, there are many different machine learning algorithms that can tease out the correlations across multiple variables.
+
+One example of risk ecommerce companies face is credit card fraud with stolen credit cards. When the owner of the card sees charges they never authorized on their monthly statement, they'll report these to the credit card company, and the charges will be reversed. The ecommerce company will lose the merchandise as well as shipping charges and labor costs. If a company receives too many chargebacks, not only will they incur expensive losses, but the credit card processors may remove them from the platorm, so it's important they have some certainty about the owner of the cards identity and legitimate interests.
+
+In this notebook, we'll demonstrate how a simplified ecommerce application might track customer orders, and use machine learning to detect chargeback risks in real time during checkout. The most important step in building any Machine Learning model is understanding the data. Knowing it's structure, application use, and the full meaning for the business will allow us to create meaningful features and labels for our models. In this notebook, we've included a fair bit of SQL to implement logic that would normally be written at the application layer to help you build an intuition about the domain. 
+
+Ecommerce Application Data Model
+--------------------------------
+We'll build out a simple eccomerce schema, and populate it with some example data. First, our store needs some products to sell. Products have a name, their price, and other metadata, like whether or not they are perishable goods.
+
+```sql
+CREATE TABLE products (
+  name TEXT PRIMARY KEY,
+  price MONEY,
+  perishable BOOLEAN
+);
+```
+
+```sql
+INSERT INTO PRODUCTS (name, price, perishable) 
+VALUES
+  ('1oz gold bar', '$1999.99', false),
+  ('a tale of 2 cities', '$19.99', false),
+  ('head of lettuce', '$1.99', true);
```

Comment:
Use `RETURNING *` so the user can see the result of this query without having to query the table again.